R/expect-proportion.R

```diff
@@ -75,6 +75,7 @@ expect_prop <- function(var,
                         func_desc = NULL) {
```

Comment:
I think `check_expect_data_pipe()` also needs to be added to the wrappers

Reply:
Yup definitely did, done!